### PR TITLE
Update caption to new style

### PR DESCRIPTION
**Pull request by lucymonie**
## What does this change?
This changes the caption style from 12px to 14px, and updates the icon from theme-coloured triangle to a grey camera icon.

## Why?
The change to the image caption is part of the liveblogs redesign. This work was originally part of this PR: https://github.com/guardian/dotcom-rendering/pull/3490

However, @oliverlloyd suggested making this a separate change as it affects all captions across the Guardian web, and this will give the change a bit more prominence, rather than being rolled in with a whole lot of other design changes. It will also make it easier to back out if there's an issue.

Figma: https://www.figma.com/file/mjJRSYtNuyPi1NGYNOH2ZO/DCR-Live-blogs?node-id=24%3A2015

### Before
Screen Shot 2021-10-20 at 15 00 39

### After
Screen Shot 2021-10-20 at 16 08 41

### CODE REVIEW

File: dotcom-rendering/src/web/lib/decidePalette.ts [hunk 1]
```diff
@@ -815,6 +819,7 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 			blockquote: textBlockquote(format),
 			numberedTitle: textNumberedTitle(format),
 			numberedPosition: textNumberedPosition(),
+			overlayed: textOverlayed(),
```
[the remark this answers is not in the document]
Comment (lucymonie) [reply dotcom-rendering/src/web/lib/decidePalette.ts:822]: Yes, good point. I think `overlayedCaption` is clearest.

**Comment by nitro-marky** [comment]
Looks great, might be good to remove those video tests from this PR

**Comment by nitro-marky** [comment]
Looks great, might be good to remove those video tests from this PR

File: dotcom-rendering/src/web/components/Caption.tsx [hunk 2]
```diff
@@ -20,10 +21,10 @@ type Props = {
 };
 
 const captionStyle = (palette: Palette) => css`
-	${textSans.xxsmall()};
+	${textSans.xsmall()};
+	line-height: 135%;
 	padding-top: 6px;
-	${textSans.xxsmall()};
-	word-wrap: break-word;
+	word-wrap: break-all;
```
[the remark this answers is not in the document]
Comment (lucymonie) [reply dotcom-rendering/src/web/components/Caption.tsx:26]: Just to illustrate, before the change, the caption was breaking like this:
Screen Shot 2021-11-02 at 15 46 14

And after the change, it looked like this:
Screen Shot 2021-11-02 at 15 47 21

File: dotcom-rendering/src/web/components/Caption.tsx [hunk 3]
```diff
@@ -107,9 +112,14 @@ const hideIconBelowLeftCol = css`
 	}
 `;
 
-const iconStyle = (palette: Palette) => css`
-	fill: ${palette.fill.captionTriangle};
-	padding-right: 3px;
+const iconStyle = css`
+	fill: ${neutral[46]};
```
[the remark this answers is not in the document]
Comment (lucymonie) [reply dotcom-rendering/src/web/components/Caption.tsx]: I have been working on this with @DavidLawes and he suggested adapting this to work with the decidePalette pattern, by adding a fill function that calls textCaption so the two end up the same colour without duplicating the functionality.

**Review by oliverlloyd — APPROVED** [review]

File: dotcom-rendering/src/web/components/Caption.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/Caption.tsx]: I don't have especially strong opinions on the naming convention `thing.text.place` or `thing.fill.name`. I think it's good to strive to make things readable but it's not worth worrying about too much, there will always be trade offs.

That said, there's a fair bit of thought behind the convention, it's [well documented here](https://docs.google.com/document/d/1nkntFJUunqrwNYszTrc2RBfyYUd1bEAeQh76yVFOLjo/edit#heading=h.odxpuxcu9hp2) if you're interested in a bit of a read

**Comment by rcrphillips** [comment]
Lovely!

**Review by OllysCoding — APPROVED** [review]
Looks good!!

**Pull request merged**